alignak/http_daemon.py

```diff
@@ -243,6 +244,10 @@ def get_socks_activity(self, socks, timeout):
     # We are asking us to stop, so we close our sockets
     def stop(self):
         self.stop_requested = True
+        for t in self.threads:
+            while t.is_alive():
+                t.join(5)
```

Comment:
I agree,

more specifically it will be stuck if any client request is actually, for whatever reason, blocked / suspended.

I'd more safely do : 
```
end = time.time() + 15    # say give 15 seconds to wait all of them
while time.time() < end: 
    for t in tuple(self.threads):
        if not t.is_alive():
            t.join()
            self.threads.remove(t)
        if not self.threads:
            break
        time.sleep(1)
```